openreview/conference/helpers.py

```diff
@@ -434,6 +434,8 @@ def get_submission_stage(request_forum, venue):
     else:
         withdraw_submission_exp_date = None
 
+    commitments_venue = request_forum.content.get('commitments_venue', 'No') == 'Yes'
```

Comment:
can we call it `ARR_commitments_venue`?

Reply:
is it possible that we would receive commitments from other venues?

Reply:
possibly from EACL (which themselves are ARR commitments)

Reply:
should we change this to `ARR_commitments_venue`? It does seem like all commitments we are getting so far come from ARR.

Reply:
sorry, I saw this too late. We can keep it as it is for now.